src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -886,6 +906,85 @@ see the [claims](#claims) section for all the options.
 See [ACME Basics](/docs/step-ca/acme-basics)
 for more guidance on configuring and using the ACME protocol with `step-ca`.
 
+#### ACME for Device Attestation
+
+<Alert severity="warning">
+  <div> 
+    This challenge type must be explicitly enabled in an ACME provisioner.
+  </div>
+</Alert>
+
+The `device-attest-01` challenge supports device attestations from iOS, iPadOS, tvOS, and YubiKeys. An attestation certificate is used to satisfy the ACME challenge. The CSR may include a device ID in a `permanentIdentifier` SAN, depending on the application. 
```

Comment:
Should this include/link to some info about this SAN? It's not very well-known nor used often.